src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -30,7 +31,7 @@ import { AssertedGenesAdditionalFieldData } from "../../components/AssertedGenes
 import { EvidenceCodesAdditionalFieldData } from "../../components/EvidenceCodesAdditionalFieldData";
 import { WithAdditionalFieldData } from "../../components/WithAdditionalFieldData";
 import { GeneticModifierAdditionalFieldData } from "../../components/GeneticModifierAdditionalFieldData";
-import ErrorBoundary from "../../components/Error/ErrorBoundary";
```

Comment:
The ErrorBoundary component would still need to be in place in order to have Error Boundaries appear in the form if there is an error. For example, if one of the form's autocompletes attempts to display a field that isn't in the api response.

Reply:
I will rebase it to alpha and check for these changes.

Reply:
Please check for errorboundry component.

Reply:
@kthorat-prog Looks good to me!